src/Amazon.Lambda.Tools/Commands/UpdateFunctionConfigCommand.cs

```diff
@@ -305,6 +328,77 @@ public async Task<GetFunctionConfigurationResponse> GetFunctionConfigurationAsyn
             }
         }
 
+        /// <summary>
+        /// Create an UpdateFunctionConfigurationRequest for the current configuration to be used to revert a failed configuration update.
+        /// </summary>
+        /// <param name="existingConfiguration"></param>
+        /// <returns><see cref="UpdateFunctionConfigurationRequest"/></returns>
+        private UpdateFunctionConfigurationRequest CreateRevertConfigurationRequest(GetFunctionConfigurationResponse existingConfiguration)
```

Comment:
Comparing this to the API docs, the only gap I see is [FileSystemConfigs](https://docs.aws.amazon.com/lambda/latest/dg/API_UpdateFunctionConfiguration.html#SSS-UpdateFunctionConfiguration-request-FileSystemConfigs), is that okay to omit?

Reply:
This property is not updated by our tooling, so there's no reason to include it in the revert method.

Reply:
Ah, good point, thanks.